ffd_info_exchange/fafsa/static/assets/css/main.css

```diff
@@ -118,10 +135,10 @@ label {
     color: #ffffff;
     display: block;
     font-size: 14px;
-    left: 50%;
+    left: 20px;
```

Comment:
It's been a minute since I did frontend for mobile devices. What's the rationale for using pixels here instead of relative units?